README.md

```diff
@@ -96,6 +96,21 @@ Verify local server TLS certificate.
 
 Accept cloudflare TOS by default.
 
+### `extraArgs`
+
+- Default: ""
+
+Extra CLI arguments to provide to cloudflared.
+
+Example:
+
+```ts
+const tunnel = await startTunnel({
+  port: 3000,
+  extraArgs: "--no-autoupdate"
```

Comment:
Maybe this could be an array [similar to how e.g. puppeteer handles it](https://github.com/puppeteer/puppeteer/blob/main/docs/troubleshooting.md#setting-up-chrome-linux-sandbox)?

Reply:
@TheAlexLichter Changed it to array